scripts/figures/graph_mode-share_sustainable.R

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8-unix -*-
+library(here)
+library(tidyverse)
+library(sf)
+source(here::here("scripts", "basemap", "functions_map.R"), encoding = "utf-8")
+
+
+# Data --------------------------------------------------------------------
+
+results <- here::here("data",
+                      "helmet_4.0.4_2018_results",
+                      "origins_demand.txt") %>%
+  readr::read_tsv(
+    col_names = c("zone", "car", "transit", "bike", "walk"),
+    col_types = "idddd",
+    skip = 1
+  ) %>%
+  dplyr::select(zone, car, transit, bike, walk) %>%
+  tidyr::pivot_longer(cols = -zone, names_to = "mode", values_to = "demand") %>%
+  dplyr::mutate(mode = factor(mode, levels = c("car", "transit", "bike", "walk"), labels = c("Henkilöauto", "Joukkoliikenne", "Pyöräily", "Kävely")))
```

Comment:
Make some kind of dictionary lookup for translations to Finnish?

Reply:
Awesome, I added this: https://github.com/HSLdevcom/mal-effect-calculations/pull/16/commits/f80fc1d4d05dfe75d5b8437f6bd3509d384185c0 I am still considering whether `utilities` folder is a good place or should I add everything to `data`... But I will change that later if needed.